macros/measures.sql

```diff
@@ -173,16 +196,6 @@ case when count(distinct {{ adapter.quote(column_name) }}) = count(*) then 1 els
 
 {%- endmacro -%}
 
-{%- macro athena__measure_median(column_name, data_type, cte_name) -%}
```

Comment:
Why is this removed?

Reply:
This seems to be sync issue; I will correct it.